cg-spec/editors_draft.html

```diff
@@ -404,11 +407,23 @@ <h2>Triples and occurrences</h2>
         -->
       </pre>
 
-    </section>
+      <p>With this solution RDF-star takes three statements to annotate an occurrence. In contrast to RDF-star's approach RDF standard reification is defined as referring to occurrences. It is however underspecified insofar as it doesn't provide a way to denote the location - some graph, document or other source of a set of triples - in which a statement occurs, as the ":in" property in the example above does. RDF standard reification needs three statements to describe a triple and would need a forth statement to specify its location. Only then a fifth statement would be able to annotate such an occurrence. Note that the ":occurrenceOf" and ":in" properties in the example above serve only illustrative purposes as standardizing a proper vocabulary to denote occurrences is still an <a href="#rdf-star-vocabulary">open issue</a>. By comparison RDF/XML's <a href="https://www.w3.org/TR/rdf-syntax-grammar/#section-Syntax-reifying">ID</a> attribute implements reification without any deviation from RDF's abstract syntax and semantics and enables statement annotation without the need for any extra statements.</p>
```

Comment:
There'll probably be other changes before we can get to punctuation, but thanks!

Reply:
There's no reason not to correct punctuation as it stands (which I tend to do to make things more readable for myself, ad also because it often *matters* to what's being said).

Reply:
You're right!